logicrunner/result_matcher.go

```diff
@@ -127,6 +135,11 @@ func (rm *resultsMatcher) isStillExecutor(ctx context.Context, object insolar.ID
 func (rm *resultsMatcher) Clear() {
 	rm.lock.Lock()
 	defer rm.lock.Unlock()
+
 	rm.executionNodes = make(map[insolar.Reference]insolar.Reference)
+
+	for reqRef := range rm.unwantedResponses {
+		log.Warn("not claimed response to request ", reqRef.String(), ", not confirmed pending?")
```

Comment:
Use `inslogger`

Reply:
no context to use inslogger :)

Reply:
and I'm using, but it's global :(

Reply:
May be add context to params? It will show pulse number, it can be useful